finetune/lm_multipurpose.py

```diff
@@ -10,9 +11,19 @@ def __init__(self, autosave_path, verbose=True):
 
     def _text_to_ids(self, *Xs, max_length=None):
         max_length = max_length or self.hparams.max_length
-        question_answer_pairs = self.encoder.encode_multi_input(*Xs, max_length=max_length)
-        tokens, mask = self._array_format(question_answer_pairs)
-        return tokens, mask
+        if type(Xs[0][0]) == str:
```

Comment:
Don't we have a target_type variable that we can use to infer behavior here?

Reply:
Or could we pull this out into an `infer_target_type` call or something similar?